docs/developers/environment.md

```diff
@@ -13,86 +13,85 @@ This is just how I work on phpVMS, to quickly reset the database and test/develo
 If you're on Windows, I highly recommend installing WSL with your distro of choice (I use Ubuntu). If you don't have `make` available, open the `Makefile` and look for the commands to run for the individual commands of what you're trying to run.
 :::
 
-### Creating and Resetting the Environment
+:::info
+You may want to disable caching during development phases to ensure your changes are applied correctly, as caching cna prevent changes from taking effect. 
+To do this, set `CACHE_DRIVER=null` in your `.env` file.
+
+:::
 
-:::caution
-This requires sqlite to be installed on your system
+:::info
+We strongly recommend reading the [Laravel Documentation](https://laravel.com/docs/11.x), as it's more detailed and up-to-date. Since phpVMS is based on Laravel, the installation process is the same.
```

Comment:
We are still using Laravel 10, I think this should point the correct version until we switch.